src/runner/microbench_runner.cpp

```diff
@@ -26,10 +26,15 @@ int main(int argc, char* argv[])
     initLogging();
 
     if (argc < 3) {
-        SPDLOG_ERROR("Usage: microbench_runner <infile> <outfile>");
+        SPDLOG_ERROR("Usage: microbench_runner <infile> <outfile> [--sgx]");
```

Comment:
Doing `--` arguments like this is misleading as it's just a string and not actually an order-insensitive flag. There are a few ways to achieve the same effect: we could use an environment variable as mentioned above,
we could remove the `--` and use the third optional string for more general flags, we could add a `mode` or list of flags to the `microbench_runner` file format, or we could do proper commandline args with a library like Boost [program options](https://www.boost.org/doc/libs/1_78_0/doc/html/program_options.html).

In this case I think the `sgx` VM is the cleanest, and if that's not possible to that, a list of colon-separated flags could be passed in the file format, e.g.

```
demo,echo,10,wamr:sgx
```